ibis/backends/base/sqlglot/rewrites.py

```diff
@@ -127,5 +132,11 @@ def sqlize(node):
         | filter_to_select
         | sort_to_select
     )
+    if DEBUG:
+        print("--------- STEP 1 ---------")  # noqa: T201
```

Comment:
Please remove these print statements which I added :D

We should have a config option at some point to help us debug the rewrite phases.

Reply:
Done!